website/client/components/notifications.vue

```diff
@@ -313,6 +346,34 @@ export default {
     document.removeEventListener('keydown', this.checkNextCron);
   },
   methods: {
+    showDeathModal () {
+      this.playSound('Death');
+      this.$root.$emit('bv::show::modal', 'death');
+    },
+    showNotificationWithModal (type, forceToModal) {
+      const config = NOTIFICATIONS[type];
+
+      if (!config) {
+        return;
+      }
+
+      if (config.achievement) {
+        this.playSound('Achievement_Unlocked');
+      } else if (config.sound) {
+        this.playSound(config.sound);
+      }
+
+      if (forceToModal) {
+        this.$root.$emit('bv::show::modal', config.modalId);
+      } else {
+        this.text(config.label(this.$t), () => {
+          this.$root.$emit('bv::show::modal', config.modalId);
+        }, false);
+      }
+    },
+    debounceCheckUserAchievements: debounce(function debounceCheck () {
+      this.checkUserAchievements();
```

Comment:
why is this debounced?

Reply:
the checkUserAchievements was called multiple times, so multiple dialogs opened + sound was played so to prevent that I debounced the call